src/components/Navbar/styles.module.scss

```diff
@@ -0,0 +1,10 @@
+.sticky {
+  @media (max-width: 599px) {
```

Comment:
how does this look on tablet / different screen sizes? Let's test the full gamut this time 😂

Reply:
This only applies on mobile viewports, anything greater won't apply the `position: sticky`